code/src/components/GameQuestions.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { faArrowLeftLong } from '@fortawesome/free-solid-svg-icons';
+import games, { GenerateQuestion } from '../reducers/games';
+import { DirectionButton, GoBackButton, ResetButton, StyledIcon } from './GlobalStyling';
```

Comment:
Nice way of solving the different styles in a GlobalStyling-file and imported it here.